stdlib/3/datetime.pyi

```diff
@@ -197,7 +197,7 @@ class datetime:
     def timetz(self) -> _time: ...
     def replace(self, year: int = ..., month: int = ..., day: int = ..., hour: int = ...,
                 minute: int = ..., second: int = ..., microsecond: int = ..., tzinfo:
-                Union[_tzinfo, bool] = ...) -> datetime: ...
+                Union[_tzinfo, bool, None] = ...) -> datetime: ...
```

Comment:
I don't remember. It might have been autogenerated. Looking at the source (`datetimemodule.c`), it seem the only allowed values are `_tzinfo` and `None`. @dmoisset, could you remove the `bool`?